invenio_stats/processors.py

```diff
@@ -25,7 +25,26 @@
 
 
 def anonymize_user(doc):
-    """Preprocess an event by anonymizing user information."""
+    """Preprocess an event by anonymizing user information.
+
+    The anonymization is done by removing fields that can uniquely identify a
+    user, such as the user's ID, session ID, IP address and User Agent, and
+    hashing them to produce a ``visitor_id`` and ``unique_session_id``. To
+    further secure the method, a randomly generated 32-byte salt is used, that
+    expires after 24 hours and is discarded. The salt values are stored in
+    Redis (or whichever backend Invenio-Cache uses). The ``unique_session_id``
+    is calculated in the same way as the ``visitor_id``, but . All of these
```

Comment:
The sentence is not complete. I guess you wanted to say that we also add a timeslice of 1 hour :)